Week3/homework/step2-2.js

```diff
@@ -2,19 +2,34 @@
 
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const numbers = [];
-
   // Replace this comment and the next line with your code
   console.log(startIndex, stopIndex, threeCallback, fiveCallback, numbers);
+  for (let i = startIndex; i <= stopIndex; i++) {
+    numbers.push(i);
+  }
+  numbers.map(number => {
```

Comment:
The `.map()` array method returns a new array of 'mapped' or transformed values of the original array. But here you are not interested in creating a new array, you just want to iterate through the elements of the array, in which case a `.forEach()` or a traditional `for` loop is more appropriate. While `.map()` works here, it will raise eyebrows from other developers (including me) trying to figure out what you are 'mapping' here (nothing).